neighbour-model/src/main/java/no/vegvesen/ixn/federation/model/ServiceProvider.java

```diff
@@ -99,12 +99,23 @@ public void updateSubscriptions(Set<LocalSubscription> newSubscriptions) {
 		this.subscriptionUpdated = LocalDateTime.now();
 	}
 
+	public void updateSubscriptionWithBrokerUrl(LocalSubscription subscription, String brokerUrl) {
+		for(LocalSubscription localSubscription : subscriptions) {
+			if (localSubscription.getSelector().equals(subscription.getSelector())){
```

Comment:
Is this enough? What if we have several Local subscriptions with the same selector, one with createNewQueue and one without? And what about status?

Reply:
Added check in the if-statement to check if createNewQueue = true